CONTRIBUTING.md

```diff
@@ -155,7 +155,11 @@ In order to format the code:
 
 ### Documentation styleguide
 
-We follow [NumPy documentation guidelines](https://numpydoc.readthedocs.io/en/latest/format.html) for PyElastica documentation.
+We follow [NumPy documentation guidelines][numpydoc-guideline] for PyElastica documentation.
+If you are interested in contributing or modifying the documentation, please refer [docs-readme][docs-readme] file.
```

Comment:
Typo: should be "please refer *to the* docs-readme file"

Reply:
Fixed: [78506ca](https://github.com/GazzolaLab/PyElastica/pull/53/commits/78506ca7da513572b56fcc6040e43711981318b8)